source/particles/particle_handler.cc

```diff
@@ -1222,6 +1225,31 @@ namespace Particles
 #endif
   }
 
+  template <int dim, int spacedim>
+  void
+  ParticleHandler<dim, spacedim>::update_ghost_particles()
+  {
+    // Nothing to do in serial computations
+    const auto parallel_triangulation =
+      dynamic_cast<const parallel::TriangulationBase<dim, spacedim> *>(
+        &*triangulation);
```

Comment:
```suggestion
    const auto parallel_triangulation =
      dynamic_cast<const parallel::TriangulationBase<dim, spacedim> *>(
        triangulation->get());
```

Reply:
This does not seem to work. No member get in dealii::Triangulation<1,1> ?

Reply:
Oh, triangulation is *SmartPoint*. Just ignore my comments ;)